includes/class-wc-payments-payment-method-messaging-element.php

```diff
@@ -48,9 +48,19 @@ public function __construct( WC_Payments_Account $account, $gateway ) {
 	 */
 	public function init(): string {
 		global $product;
-		$price           = $product->get_price();
-		$currency_code   = get_woocommerce_currency();
-		$billing_country = WC()->countries->get_base_country();
+		$billing_country    = WC()->countries->get_base_country();
+		$currency_code      = get_woocommerce_currency();
+		$base_product_price = WC_Payments_Utils::prepare_amount( $product->get_price(), $currency_code );
+		$all_variation_ids  = $product->get_children();
+
+		$variations_price_list = [ 'base_product' => $base_product_price ];
```

Comment:
`$product_prices` would be a better name here.

Reply:
I have updated this here - e451407 . Thanks.